### DISCUSSION TITLE
Webradios

### DISCUSSION THREAD

**Discussion by lokapal** [jcorporation/myMPD#990] [Q&A]
Could you somehow clarify how to add to mympd A LOT OF webradios, please?
I'd like to have possibility to have the local json collections, groups etc. As it was in mympd 3.xxx
`curl -s -H 'Accept: application/json' https://somafm.com/channels.json >/tmp/somafm.json`
`wget http://radcap.ru/index-db.html`
All radios that are included to AlbumPlayer [https://albumplayer.ru/linux/](url), especially FLAC collection.

E.g. if we have local files mympd can open and play .pls files (still no logos are shown), but it cannot play .m3u at all. What formats are supported? Where to put webradio logo to have it displayed? What format is the best for it? Could it be clarified in docs?

P.S. I use myMPD 9.5.4 at archphile-based RPI3 distribution, I can re-compile it with any options by your advice.

**Comment by jcorporation** [comment]
It is really simple. There are two directories:
- /var/lib/mympd/webradios: add here extm3u files
- /var/lib/mympd/pics/thumbs: for stream images

Naming convention is described here: https://jcorporation.github.io/myMPD/references/pictures

Have a look at webradiodb for examples and format definition of extm3u files for myMPD: https://github.com/jcorporation/webradiodb

P.S. this myMPD version is out of date, 10.2.4 is the current stable release.

**Comment by lokapal** [comment]
10.2.4 doesn't update webradios from your database at all (in archphile distribution, it's frozen at binaries/libraries from 2018.07). Moreover, I can't recompile mpd, so have to stick to MPD 0.22 that is built-in.
If I'll put to webradios A FEW THOUSAND of radios it will definitely render "favourites" feature unusable (do you support "the most favourites" ;-) ? Are there any possibility to feed to myMPD the additional local JSONs with webradios databases?

**Comment by jcorporation** [comment]
> Are there any possibility to feed to myMPD the additional local JSONs with webradios databases?

No, the webradiodb uri is hardcoded, but you can add the webradios to the official webradiodb project.

> If I'll put to webradios A FEW THOUSAND of radios it will definitely render "favourites" feature unusable (do you support "the most favourites" ;-)

Why do you assume this?
